fastlane_bot/data/multichain_addresses.csv

```diff
@@ -49,7 +49,6 @@ sushiswap_v3,core,uniswap_v3,0xc35DADB65012eC5796536bD9864eD8773aBc74C4,N/A,,,
 fraxswap_v2,dogechain,uniswap_v2,0x67b7DA7c0564c6aC080f0A6D9fB4675e52E6bF1d,0x0f6A5c5F341791e897eB1FB8fE8B4e30EC4F9bDf,,,
 balancer,ethereum,balancer,BALANCER_VAULT_ADDRESS,0xBA12222222228d8Ba445958a75a0704d566BF2C8,0,0,
 carbon_v1,ethereum,carbon_v1,0xC537e898CD774e2dCBa3B14Ea6f34C93d5eA45e1,0xC537e898CD774e2dCBa3B14Ea6f34C93d5eA45e1,NA,17087375,
-altered_carbon,ethereum,carbon_v1,0xA31ac5a9077d8EE58F4cd0011C315552De29542B,0xA31ac5a9077d8EE58F4cd0011C315552De29542B,NA,19375311,

```

Comment:
How is this change related here?

Reply:
not a supported exchange so it breaks the use of carbon_v1_forks on ethereum - should be deleted